src/LeftNav/right-arrow.svg

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
should make this a `RightArrow` component with svg content (e.g. https://github.com/Clever/apps-dashboard/blob/master/pages/ui/components/TopBar/Logo.jsx)

would mean that clients don't have to copy `right-arrow.svg` to the right place on the server in order to use the LeftNav